WcaOnRails/app/models/post_tag.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class PostTag < ApplicationRecord
+  belongs_to :post
+
+  before_validation { self.tag = self.tag.strip }
+
+  validates :tag, format: { with: /\A[a-zA-Z]+\z/, message: "only allows letters" }
```

Comment:
Yeah, supporting letters sound very reasonable! I was trying something new here: start strict and become less strict as people ask for it. That's easier than the other direction, where you have to write migrations to handle substituting characters that have suddenly become illegal.